src/utilities.cpp

```diff
@@ -96,29 +101,18 @@ std::string convertToString(double value, bool fullPrecision)
     return strs.str();
 }
 
-int convertToInt(const std::string &in, bool *ok)
+bool convertToInt(const std::string &in, int *out)
```

Comment:
We should pass this int by reference as we have done for the `convertToDouble` function.

Reply:
301fb571